From 9038acaf742cddf23f5e725bf46100fdfbbc9aa7 Mon Sep 17 00:00:00 2001
From: Morten Gjerding <mogje@fysik.dtu.dk>
Date: Thu, 18 Nov 2021 15:00:44 +0100
Subject: [PATCH] Compatiblity fix for matplotlib 3.5.0

---
 ase/dft/bz.py | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/ase/dft/bz.py b/ase/dft/bz.py
index b23cd919a..ff96044c7 100644
--- a/ase/dft/bz.py
+++ b/ase/dft/bz.py
@@ -53,6 +53,14 @@ def bz_plot(cell, vectors=False, paths=None, points=None,
                 self.set_positions((xs[0], ys[0]), (xs[1], ys[1]))
                 FancyArrowPatch.draw(self, renderer)
 
+            # FIXME: Compatibility fix for matplotlib 3.5.0: Handling of 3D
+            # artists have changed and all 3D artists now need
+            # "do_3d_projection". Since this class is a hack that manually
+            # projects onto the 3D axes we don't need to do anything in this
+            # method.
+            def do_3d_projection(self, *_, **__):
+                return 0
+
         azim = pi / 5
         elev = elev or pi / 6
         x = sin(azim)
-- 
GitLab

