diff -ru ase-3.13.0/ase/calculators/vasp.py ase-3.13.1/ase/calculators/vasp.py
--- ase-3.13.0/ase/calculators/vasp.py	2017-01-25 13:04:06.000000000 +0100
+++ ase-3.13.1/ase/calculators/vasp.py	2017-02-27 15:42:54.049118533 +0100
@@ -1202,7 +1202,7 @@
     def write_potcar(self, suffix=""):
         """Writes the POTCAR file."""
         import tempfile
-        potfile = open('POTCAR' + suffix, 'wb')
+        potfile = open('POTCAR' + suffix, 'w')
         for filename in self.ppp_list:
             if filename.endswith('R'):
                 for line in open(filename, 'r'):
